src/SymbolCollector.Android/MainActivity.cs

```diff
@@ -354,6 +354,11 @@ public MainActivity()
                     o.BlackListedPaths.Add("/system/vendor/bin/netstat");
                     o.BlackListedPaths.Add("/system/vendor/bin/swapoff");
                 });
+                c.AddOptions().Configure<ObjectFileParserOptions>(o =>
+                {
+                    o.IncludeHash = true; // Backing store sorted format does not support hash distinction yet.
```

Comment:
We can turn it off later for the app, once we deploy this branch and publish the server support live